spiped/main.c

```diff
@@ -27,6 +28,12 @@ usage(void)
 	exit(1);
 }
 
+static void
+signal_handler(int signo)
+{
+	exit(0);
```

Comment:
Style requires a blank line separating the (in this case empty) local variable declarations from the code in the function.